src/main/java/io/getunleash/metric/UnleashMetricServiceImpl.java

```diff
@@ -28,6 +38,7 @@ public UnleashMetricServiceImpl(
         this.started = LocalDateTime.now(ZoneId.of("UTC"));
         this.unleashConfig = unleashConfig;
         this.metricSender = metricSender;
+        this.maxInterval = Integer.max(20, 300 / Integer.max(Long.valueOf(unleashConfig.getSendMetricsInterval()).intValue(), 1));
```

Comment:
Happy to discuss this, our goal was that the max should equal 300 seconds at most. because 5 minute interval seems to be enough. I realize the metric sending could be bigger. Suggestions?